VERSION

```diff
@@ -1 +1 @@
-v5.0.0
\ No newline at end of file
+v5.1.0-alpha
```

Comment:
5.0.1 would be better for a minor releases?

Reply:
We have few issues which need to fix in 5.0.1, and 5.1.0 is for next quarter release.